src/components/v2v3/shared/SplitList.tsx

```diff
@@ -2,64 +2,81 @@ import { BigNumber } from '@ethersproject/bignumber'
 import { Space } from 'antd'
 import { Split } from 'models/splits'
 import { useMemo } from 'react'
-import { getProjectOwnerRemainderSplit } from 'utils/splits'
-import { SplitItem } from './SplitItem'
+import {
+  getProjectOwnerRemainderSplit,
+  processUniqueSplits,
+} from 'utils/splits'
+import { SplitItem, SplitProps } from './SplitItem'
 
 export default function SplitList({
   splits,
-  showSplitValues = false,
+  oldSplits,
+  showAmounts = false,
   showFees = false,
   currency,
   totalValue,
   projectOwnerAddress,
   valueSuffix,
   valueFormatProps,
   reservedRate,
+  showDiffs,
 }: {
   splits: Split[]
+  oldSplits?: Split[]
   currency?: BigNumber
   totalValue: BigNumber | undefined
   projectOwnerAddress: string | undefined
-  showSplitValues?: boolean
+  showAmounts?: boolean
   showFees?: boolean
   valueSuffix?: string | JSX.Element
   valueFormatProps?: { precision?: number }
   reservedRate?: number
+  showDiffs?: boolean
 }) {
   const ownerSplit = useMemo(() => {
     if (!projectOwnerAddress) return
     return getProjectOwnerRemainderSplit(projectOwnerAddress, splits)
   }, [projectOwnerAddress, splits])
 
+  const oldOwnerSplit = useMemo(() => {
+    if (!oldSplits || !projectOwnerAddress || !showDiffs) return
+    return getProjectOwnerRemainderSplit(projectOwnerAddress, oldSplits)
+  }, [projectOwnerAddress, oldSplits, showDiffs])
+
+  const uniqueSplits = processUniqueSplits({
+    oldSplits: showDiffs ? oldSplits : undefined,
```

Comment:
**suggestion** i think we should include a `showDiff` prop in `SplitItem` instead. This means we can be explicitly clear about when we want to show a diff or not for a given SplitItem.